src/main/java/org/sefglobal/scholarx/controller/mentor/UserController.java

```diff
@@ -0,0 +1,60 @@
+package org.sefglobal.scholarx.controller.mentor;
+
+import org.sefglobal.scholarx.exception.ResourceNotFoundException;
+import org.sefglobal.scholarx.exception.UnauthorizedException;
+import org.sefglobal.scholarx.model.Mentee;
+import org.sefglobal.scholarx.model.Mentor;
+import org.sefglobal.scholarx.service.MenteeService;
+import org.sefglobal.scholarx.service.MentorService;
+import org.sefglobal.scholarx.util.EnrolmentState;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@RequestMapping("/program")
+public class UserController {
```

Comment:
Can't we use the existing `programController` instead of the new class.

Reply:
Without creating `controller/mentor` directory?